python/openinference-semantic-conventions/src/openinference/semconv/trace/__init__.py

```diff
@@ -148,6 +149,39 @@ class MessageAttributes:
     """
 
 
+class MessageContentAttributes:
+    """
+    Attributes for the contents of user messages sent to an LLM
+    """
+
+    MESSAGE_CONTENT_TYPE = "message_content.type"
+    """
+    The type of the content, such as "text" or "image_url".
+    """
+    MESSAGE_CONTENT_TEXT = "message_content.text"
+    """
+    The text content of the message, if the type is "text".
+    """
+    MESSAGE_CONTENT_IMG = "message_content.image"
+    """
+    The image content of the message, if the type is "image_url".
+    An image can be made available to the model by passing a link to
+    the image or by passing the base64 encoded image directly in the
+    request.
+    """
+
+
+class ImageAttributes:
+    """
+    Attributes for image contents in user messages to an LLM
+    """
+
+    IMAGE_URL = "url"
```

Comment:
shouldn't this have the `image` prefix?
```suggestion
    IMAGE_URL = "image.url"
```

Reply:
If we add the prefix it will look as:
`llm.input_messages.0.message.message_content.image.image.url` no?